fuseimg/data/ops/aug/geometry.py

```diff
@@ -219,3 +219,39 @@ def __call__(self, sample_dict: NDict, key: str, axis_squeeze: int, channels: in
         
         sample_dict[key] = aug_output
         return sample_dict
+
+
+class OpResizeTo(OpBase):
+    """
+    Resizes an image into the given dimensions. Currently supports only ndarray
+    """
+    def __init__(self, channels_first: bool):
+        """
+        :param channels_first: assign True iff the input is in CxHxW format.
```

Comment:
Can we write the code so any KD (3D, 4D, ...) will be supported?

Reply:
@mosheraboh 

I changed it to supports k dimensions but I'm not that happy with the current solution:
1. I believe there is a simple one to permute the axis in the desired way, tho I didn't find something else then give the desired permutation to `np.transpose()`
2. In the current solution in each sample we are doing the same computation of the permutation. So I think it is preferred to be done in the `__init__` function. But then we need to supply the number of dimensions in the pipeline.

What do you think? :)